site/docs/about/glossary.mdx

```diff
@@ -36,3 +36,11 @@ Salt is the J.P. Morgan design system, an open-source solution for building exce
 Salt is under active development. At this stage, we don't recommend theming unless absolutely necessary.
 
 For an overview of theming, refer to our [theming documentation](/salt/themes/index).
+
+## Disabled state
+
+A component in a disabled state isn't considered relevant to the user’s workflow at that time. As the function isn't available to the user, the component doesn't need to meet minimum contrast requirements. Consider if a disabled state is the right UI pattern, or if a progressive reveal might be more appropriate. Salt doesn't offer error, warning, or success states for disabled components, as these imply required user action; consider using a read-only state or progressive reveal instead. Some components have exceptions, such as Button, so it can be discovered where context is important.
+
+## Read-only state
+
+A component in a read-only state is considered relevant to the user’s workflow, allowing users to view and interact with the content without modifying it. It must meet accessibility requirements, including minimum contrast. The system supports status messages for read-only components, as RAG states may require visibility for recognition and further action. Consider if a read-only state is the right UI pattern, or if a disabled state might be more appropriate when modification is not needed.
```

Comment:
```suggestion
A component in a read-only state is considered relevant to the user’s workflow, allowing users to view and interact with the content without modifying it. It must meet accessibility requirements, including minimum contrast. The system supports validation states (error, warning or success) for read-only components, as RAG states may require visibility for recognition and further action. Consider if a read-only state is the right UI pattern, or if a disabled state might be more appropriate when modification is not needed.
```
status messages seems too narrow?
This copy introduces the term RAG states which isn't previously defined and I'm not sure how well known it is.
The last sentence seems wrong, disabled is very different non-modifiable which is what read-only is.